docs/static/releasenotes.asciidoc

```diff
@@ -11,6 +12,49 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-1-1]]
+=== Logstash 6.1.1 Release Notes
+*  There are no user-facing changes in Logstash core in this release.
+
+[float]
+==== Input Plugins
+
+*`Beats`*:
+
+* Fix issue with close_wait connections, making sure keep alive sent back to the client. (https://github.com/logstash-plugins/logstash-input-beats/pull/272[#272])
+
+*`HTTP`*:
+
+* In the event that all webserver threads are busy this plugin will now return a 429, busy, error. (https://github.com/logstash-plugins/logstash-input-http/pull/75[#75])
+
+*`JDBC`*:
+
+* Fix connection and memory leak. (https://github.com/logstash-plugins/logstash-input-jdbc/issues/251[#251])
+
+*`Syslog`*:
+
+* Fix issue where stopping a pipeline. (e.g., while reloading configuration) with active inbound syslog connections could cause Logstash to crash. (https://github.com/logstash-plugins/logstash-input-syslog/issues/40[#40])
+
+[float]
+==== Filter Plugins
+
+*`Split`*:
+
+* Fix crash on arrays with null values. (https://github.com/logstash-plugins/logstash-filter-split#31[#31])
+
+[float]
+==== Codecs
+
+*`Line`*:
+
+* Fixed thread safety issue. (https://github.com/logstash-plugins/logstash-codec-line/pull/13[#13])
+
+*`Netflow`*:
+
+* Added vIPtela support.
+* Added fields for Cisco ASR1k.
+
```

Comment:
The entries should all be the same tense. Previous ones used present tense, these use past tense. I'd be inclined to switch all of them to past tense, because "Fix thread safety issue" seems weird.